sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -62,6 +62,7 @@ export interface CreateQueueOptions extends OperationOptions {
     defaultMessageTimeToLive?: string;
     duplicateDetectionHistoryTimeWindow?: string;
     enableBatchedOperations?: boolean;
+    enableExpress?: boolean;
```

Comment:
Looks like the build has failed because of the outdated "API Report".

![image](https://user-images.githubusercontent.com/10452642/92313981-986a9780-ef86-11ea-9f39-cb93ba6b9a29.png)


Please update the `api.md` file by building the service-bus project again and commit the changes. :)

Reply:
@HarshaNalluru thanks for the feedback. I have checked-in the service-bus.api.md file